src/main/resources/BridgeServer2.conf

```diff
@@ -101,58 +99,38 @@ upload.cms.certificate.organization = Sage Bionetworks
-
-// Maximum 100 MB per zip entry
+
+# Maximum 100 MB per zip entry
 max.zip.entry.size = 100000000
-// Maximum 100 zip entries per archive
+# Maximum 100 zip entries per archive
 max.num.zip.entries = 100
 
-# AWS credentials for reading/writing consent documents to S3
-aws.key.consents = dummy-value
-aws.secret.key.consents = dummy-value
-
-# AWS credentials for sending push notifications
-sns.key = dummy-value
-sns.secret.key = dummy-value
-
 # Buckets for the content of each consent revision
-local.consents.bucket = org-sagebridge-consents-local
-dev.consents.bucket = org-sagebridge-consents-dev
-uat.consents.bucket = org-sagebridge-consents-uat
-prod.consents.bucket = org-sagebridge-consents-prod
+local.consents.bucket = org-sagebridge-consents-${bucket.suffix}
 
 # Bridge Exporter SQS queues
+
+exporter.request.sqs.queue.url=https://sqs.us-east-1.amazonaws.com/649232250620/Bridge-EX-Request-${bucket.suffix}
+# Until we have a SQS intializer, we must break the suffix pattern
 local.exporter.request.sqs.queue.url=https://sqs.us-east-1.amazonaws.com/649232250620/Bridge-EX-Request-local
-dev.exporter.request.sqs.queue.url=https://sqs.us-east-1.amazonaws.com/649232250620/Bridge-EX-Request-dev
-uat.exporter.request.sqs.queue.url=https://sqs.us-east-1.amazonaws.com/649232250620/Bridge-EX-Request-uat
-prod.exporter.request.sqs.queue.url=https://sqs.us-east-1.amazonaws.com/649232250620/Bridge-EX-Request-prod
 
 # Bridge User Data Download Service SQS queues
+udd.sqs.queue.url = https://sqs.us-east-1.amazonaws.com/649232250620/Bridge-WorkerPlatform-Request-${bucket.suffix}
 local.udd.sqs.queue.url = https://sqs.us-east-1.amazonaws.com/649232250620/Bridge-WorkerPlatform-Request-local
-dev.udd.sqs.queue.url = https://sqs.us-east-1.amazonaws.com/649232250620/Bridge-WorkerPlatform-Request-dev
-uat.udd.sqs.queue.url = https://sqs.us-east-1.amazonaws.com/649232250620/Bridge-WorkerPlatform-Request-uat
-prod.udd.sqs.queue.url = https://sqs.us-east-1.amazonaws.com/649232250620/Bridge-WorkerPlatform-Request-prod
 
 # List of apps that should never be deleted
-local.app.whitelist = api
-dev.app.whitelist = api
-uat.app.whitelist = api,ios-sdk-int-tests,asthma,cardiovascular
-prod.app.whitelist = api,asthma,breastcancer,cardiovascular,diabetes,fphs,fphs-lab,ohsu-molemapper,parkinson,parkinson-lux,lilly
+local.app.whitelist = api,shared
+dev.app.whitelist = api,shared
+uat.app.whitelist = api,shared,ios-sdk-int-tests,asthma,cardiovascular
+prod.app.whitelist = api,shared,asthma,breastcancer,cardiovascular,diabetes,fphs,fphs-lab,ohsu-molemapper,parkinson,parkinson-lux,lilly
 
-local.usersigned.consents.bucket = local.usersigned.consents.bucket
+local.usersigned.consents.bucket = org-sagebridge-usersigned-consents-${bucket.suffix}
 dev.usersigned.consents.bucket = bridgepf-develop-awss3usersignedconsentsdownloadb-apwbxc8ldmj2
 uat.usersigned.consents.bucket = bridgepf-uat-awss3usersignedconsentsdownloadbucke-hcuoz4eztd8g
```

Comment:
We can change dev and staging to follow the pattern. These aren't actually created by Infra. We'd have to create new buckets for dev and staging, but with the AWS migration, that's going to happen anyway.